play/client-circe/src/main/scala/endpoints/play/client/CirceEntities.scala

```diff
@@ -2,15 +2,20 @@ package endpoints.play.client
 
 import endpoints.algebra
 import endpoints.algebra.CirceEntities.CirceCodec
+import io.circe.{Json, jawn}
+import play.api.libs.ws.{BodyWritable, InMemoryBody}
+import play.api.mvc.Codec
+import play.api.http.ContentTypes
 import endpoints.play
-import io.circe.jawn
-import endpoints.play.PlayCirce.circeJsonWriteable
 
 /**
   * Implements [[algebra.CirceEntities]] for [[play.client.Endpoints]]
   */
 trait CirceEntities extends algebra.CirceEntities { this: Endpoints =>
 
+  implicit def circeJsonBodyWriteable(implicit codec: Codec): BodyWritable[Json] =
+    new BodyWritable[Json](json => InMemoryBody(codec.encode(json.noSpaces)), ContentTypes.JSON)
```

Comment:
We used to have both `play-client-circe` and `play-server-circe` depend on a common `play-circe` artifact providing a `Writeable[Json]` instance. It seems that now, Play WS doesn’t use `Writeable` anymore? So we should probably remove this `play-circe` module?

Reply:
Your right, will migrate play-circe to play-server-circe.